### DISCUSSION TITLE
显示总数2000多，但是获取了1000多就自动停止了

### DISCUSSION THREAD

**Discussion by MiSanl** [Chilfish/Weibo-archiver#60] [Q&A]
![image](https://github.com/user-attachments/assets/a4cd1dc5-fb84-45fb-b80c-30e798f87916)
如上图，总数2592（并且这个总数爬取过程中会变），但是获取到1113后就自动停止，并弹出下载窗口了，不知道是显示的bug还是获取过于频繁被迫中止了，希望大佬可以解答一下

**Comment by Chilfish** [comment]
如果还没关闭窗口的话，可以贴一下 F12（审查元素）-> 终端（Console）的截图吗？也许有哪里报错了

**Reply by MiSanl** [reply]
那等我再爬一次，需要点时间

**Reply by MiSanl** [reply]
![image](https://github.com/user-attachments/assets/2a043838-c43c-4a09-812f-7f1968c4cf71)
又爬了一次，停止的位置是一样的，并且我半年半年爬的话，显示的总数为652+722

**Reply by MiSanl** [reply]
![image](https://github.com/user-attachments/assets/53f56b6d-992d-4365-b942-9f4fa85a1531)
设置爬取前半年的，进度是正确的，但是控制台的信息好像差不多，就是获取的页数不一样，但导出的数量是一样的？

**Reply by MiSanl** [reply]
在预览界面看，半年半年导出的内容和整年导出（1113/2592停止）的内容是一样的

**Reply by Chilfish** [reply]
ok，之后我尝试找下原因

**Comment by Chilfish** [comment]
似乎发现了原因之一，微博学推特那样，一次性搜一定数量的微博就触发限制了…… 变成超过1000条左右后就不返回数据了，程序就以为没有更多微博而结束然后导出了

下图也是导出到第50页后的样子，list 本该还有数据来着的…之前是没这个问题的。我再看看它具体的机制😅

![image](https://github.com/user-attachments/assets/d8fa52d4-354d-4724-8275-7919428f2d2d)
